<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
  * Released as open source by Gabriel Caudrelier
  *
  * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
  *
  * https://github.com/metrodango/pip3line
  *
  * Released under BSD 3-Clause see LICENSE for more information
</comment>
 <class>HexEncodeWidget</class>
 <widget class="QWidget" name="HexEncodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="normalRadioButton">
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="escapedRadioButton">
     <property name="text">
      <string>Escaped strict (all char)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="escapedMixedRadioButton">
     <property name="text">
      <string>Escaped mixed (only non-printable char)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cstyleRadioButton">
     <property name="text">
      <string>C-style array</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="csvRadioButton">
     <property name="text">
      <string>CSV</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="addPrefixCheckBox">
     <property name="text">
      <string>Add &quot;0x&quot; prefix (only applicable to normal mode)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
